cmd/whitesourceExecuteScan.go

```diff
@@ -157,6 +157,13 @@ func whitesourceExecuteScan(config ScanOptions, _ *telemetry.CustomData, commonP
 }
 
 func runWhitesourceExecuteScan(ctx context.Context, config *ScanOptions, scan *ws.Scan, utils whitesourceUtils, sys whitesource, commonPipelineEnvironment *whitesourceExecuteScanCommonPipelineEnvironment, influx *whitesourceExecuteScanInflux) error {
+	if config.BuildTool == golangBuildTool && config != nil {
```

Comment:
should we rather do the check only when `PrivateModulesGitToken` is set ? , maybe a little bit more safer and not based on `buildTool `

Reply:
done